src/tests/frontend/specs/focus.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+describe('caret focus', function () {
+  describe('doesnt focus on editor using focusOnEditor URL Param', function () {
+    before(async function () {
+      await helper.aNewPad({
+        params: {
+          focusOnEditor: 'false'
+        }
+      });
+    });
```

Comment:
Wait for postAceInit so we know there has been an opportunity to focus:

```suggestion
      const {classList} = helper.padChrome$.document.getElementById('editorcontainer');
      await helper.waitForPromise(() => classList.contains('initialized'));
    });
```

Reply:
#5195 will make this change unnecessary.

Reply:
#5195 is merged.